### ISSUE TITLE
Unable to reference security scheme defined in another file

### ISSUE THREAD

**Issue by johnmckinzie** [stoplightio/desktop#78]
I have defined a security scheme in my `common.oas2` file. I have a separate file that defines my API endpoints. When I go to the **Global Security** section in my API file, I'm unable to reference the security scheme I defined in my `common.oas2`.

**Comment by jayshao** [comment]
Security scheme is also one of the biggest items I would like to share between API definitions...

**Comment by marbemac** [comment]
Agreed this would be nice - will try to work into our December roadmap and post back here when I have a time!
